packages/circuit-ui/components/Selector/Selector.tsx

```diff
@@ -53,16 +64,18 @@ export interface SelectorProps
    * Whether the user can select multiple options.
    */
   multiple?: boolean;
-  /**
-   * The ref to the HTML DOM element
-   */
-  ref?: Ref<HTMLInputElement>;
```

Comment:
Yeah, I discovered a better way of typing this by passing the HTML element type and the prop type to `forwardRef` as generic props instead of typing the wrapped function args. So yes, we can use the same approach in other components.

Reply:
Amazing!